notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>I like looking at the flagged data, but we definitely don't want to use the KeplerQualityFlags.decode here. The QLP flags do not correspond to the Kepler flags. Instead, you should refer to the QLP Data Production Description Doc (https://archive.stsci.edu/hlsps/qlp/hlsp_qlp_tess_ffi_all_tess_v1_data-prod-desc.pdf) for what the 4096 quality flag means.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I assume the pdf listed has a typo and that low precision points should be 4096 not 4196?</p>

Reply:
<p>Ah, good catch. Yes, that is almost certainly a typo.</p>